tests/unit/workflows/test_workflows_tasks_actions.py

```diff
@@ -1124,3 +1125,63 @@ def test_validate_record_raises_when_record_is_invalid():
 
     with pytest.raises(ValidationError):
         _validate_record(obj, eng)
+
+
+@patch('inspirehep.modules.workflows.tasks.actions.get_document_in_workflow')
+@patch(
+    'inspirehep.modules.workflows.tasks.refextract.match',
+    return_value=(_ for _ in [])
```

Comment:
why not `iter([])`?

Reply:
My fault, I suggested it. But this is not really important, right? Let's try to merge this today.

Reply:
I was scratching my head for a couple of minutes trying to understand this, so I would be in favor of changing it. I agree on trying to get this merged today.

Reply:
Ok